podcasts/SFSafariViewController+Creation.swift

```diff
@@ -0,0 +1,21 @@
+import SafariServices
+
+extension SFSafariViewController {
+
+    /// Creates a SFSafariViewController with consistent config to be used in the app
+    static func controller(with url: URL, config: SFSafariViewController.Configuration = .appDefault) -> SFSafariViewController {
+        let controller = SFSafariViewController(url: url, configuration: config)
+        controller.dismissButtonStyle = .close
+
+        return controller
+    }
```

Comment:
What do you think about changing that to:

```suggestion
    convenience init(with url: URL, config: SFSafariViewController.Configuration = .appDefault) {
        self.init(url: url, configuration: config)
        dismissButtonStyle = .close
    }
```